metrics_sdk/lib/opentelemetry/sdk/metrics/view/registered_view.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module SDK
+    module Metrics
+      module View
+        # RegisteredView
+        class RegisteredView
+          attr_reader :name, :aggregation, :attribute_keys
+
+          def initialize(name, **options)
+            @name = name
+            @options = options
+            @aggregation = options[:aggregation]
+            @attribute_keys = options[:attribute_keys] || {}
+          end
+
+          def match_instrument?(metric_stream)
```

Comment:
There's one more pattern we need to consider for the spec. [If the name is `*`, then it should always match.](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk.md?plain=1#L275).

Reply:
For current implementation, the wildcard `*` should match everything.